index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
           </pre>
         </section>
       </section>
+
+      <section>
+        <h3>Data Schemas</h3>
+
+        <p>
+Data schemas are useful when enforcing a specific structure on a given
+collection of data. There are at least two types of data schemas that this
+specification considers:
+        </p>
+
+        <ul>
+          <li>
+Data verification schemas, which are used to <a>verify</a> that the structure
```

Comment:
```suggestion
Data validation schemas, which are used to <a>validate</a> that the structure
```

Reply:
See comments in https://github.com/w3c/vc-data-model/pull/1203/files#r1279865135